staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/admission.go

```diff
@@ -136,9 +137,13 @@ func (a *QuotaAdmission) Validate(ctx context.Context, attr admission.Attributes
 	if attr.GetSubresource() != "" {
 		return nil
 	}
-	// ignore all operations that are not namespaced
-	if attr.GetNamespace() == "" {
+	// ignore all operations that are not namespaced or creation of namespaces
+	if attr.GetNamespace() == "" || isNamespaceCreation(attr) {
 		return nil
 	}
 	return a.evaluator.Evaluate(attr)
 }
+
+func isNamespaceCreation(attr admission.Attributes) bool {
```

Comment:
nits:
1. put the cheap check (attr.GetOperation() == admission.Create) first
2. construct the comparison value once as a package var `var namespaceGVK = v1.SchemeGroupVersion.WithKind("Namespace").GroupKind()`

Reply:
Thanks, will change both.